streaming/stream_session.cc

```diff
@@ -45,10 +45,47 @@ static sstables::offstrategy is_offstrategy_supported(streaming::stream_reason r
     static const std::unordered_set<streaming::stream_reason> operations_supported = {
         streaming::stream_reason::bootstrap,
         streaming::stream_reason::replace,
+        streaming::stream_reason::removenode,
+        streaming::stream_reason::decommission,
+        streaming::stream_reason::repair,
+        streaming::stream_reason::rebuild,
     };
     return sstables::offstrategy(operations_supported.contains(reason));
 }
 
+class offstrategy_trigger {
+    sharded<replica::database>& _db;
+    table_id _id;
+    streaming::plan_id _plan_id;
+    replica::column_family& _cf;
+    std::chrono::steady_clock::time_point last_update;
+
+public:
+    offstrategy_trigger(sharded<replica::database>& db, table_id id, streaming::plan_id plan_id)
+        : _db(db)
+        , _id(id)
+        , _plan_id(plan_id)
+        , _cf(_db.local().find_column_family(_id))
+    {
+        _cf.enable_off_strategy_trigger();
+    }
+    void update() {
+        auto now = std::chrono::steady_clock::now();
```

Comment:
How about seastar's lowres_clock? It's even faster.

Reply:
OK. I did that in v3.